.github/workflows/test-ci.yml

```diff
@@ -189,7 +189,7 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           python -m pip install -e .[test,dev]
-          python -m pip install finufft pooch brainweb-dl torch
+          python -m pip install finufft pooch brainweb-dl torch phantominator
```

Comment:
I think we can also stick to brainweb-dl? Is there any reason why you need to use phantominator?

Reply:
I was having some SSL verification problem and decided to use this. I am trying to solve this.